akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/UntypedWatchingTypedTest.java

```diff
@@ -78,12 +78,21 @@ public Ping(ActorRef<Pong> replyTo) {
   @Test
   public void testItWorks() {
     //#create-untyped
-    ActorSystem as = ActorSystem.apply();
+    akka.actor.ActorSystem as = akka.actor.ActorSystem.create();
     akka.actor.ActorRef untyped = as.actorOf(Untyped.props());
     //#create-untyped
     TestProbe probe = new TestProbe(as);
     probe.watch(untyped);
     probe.expectTerminated(untyped, Duration.create(1, "second"));
     as.terminate();
   }
+
+  @Test
+  public void testConversionFromUnTypedSystemToTyped() {
+    //#convert-untyped
+    akka.actor.ActorSystem untypedActorSystem = akka.actor.ActorSystem.create();
+    ActorSystem<Void> typedActorSystem = Adapter.toTyped(untypedActorSystem);
+    //#convert-untyped
+    typedActorSystem.terminate();
```

Comment:
Await, to avoid leaking logging and such into next test

Reply:
or better if we have a shutdown method in TestKit

Reply:
We do :) Doh!

Reply:
but i want to show the creation of the actor system in docs so will do it manually here

Reply:
yes, but terminate is outside the doc snippet
terminate is async, returning a Future